analyzer/src/main/kotlin/managers/utils/SpdxResolvedDocument.kt

```diff
@@ -272,8 +272,8 @@ internal fun SpdxExternalDocumentReference.resolve(
             baseUri,
             createAndLogIssue(
                 source = managerName,
-                message = "'$spdxDocument' defined in '$baseUri' as part of '$externalDocumentId' is not a valid " +
-                        "URI."
+                message = "The SPDX document '$spdxDocument' defined in '$baseUri' as part of '$externalDocumentId' " +
```

Comment:
I find the resulting sentence a bit strange, since an SPDX document is not a URI. Should it better be something like "The SPDX document reference ... is not a valid URI."?

Reply:
> since an SPDX document is not a URI.

Well, the `spdxDocument` field *is* defined to be a URI according to the spec. The field name also is not called `spdxDocumentReference` or so.

But if you prefer, I could write this as "The SPDX document *at* '$spdxDocument'...".